msk/actions/create.py

```diff
@@ -257,6 +257,41 @@ def commit_changes(self):
             self.git.add('.')
             self.git.commit(message='Initial commit')
 
+    def force_push(self, get_repo_name: Callable = None) -> Optional[Repository]:
+        if ask_yes_no(
+                'Are you sure you want to overwrite the remote github repo? '
+                'This cannot be undone and you will lose your commit '
+                'history! (y/N)',
+                False):
+            repo_name = (get_repo_name and get_repo_name()) or (
+                    self.name + '-skill')
+            repo = self.user.get_repo(repo_name)
+            call(['git', 'push', '-f', 'origin', 'master'],
```

Comment:
To be consistent, this can be written as `self.git.push('origin', 'master', force=True)`